vulnerability/worker_test.go

```diff
@@ -48,7 +48,7 @@ type runnerMock struct {
 }
 
 func (s runnerMock) ScanVM(_ context.Context, opts flag.Options) (trivy_types.Report, error) {
-	writer, err := os.OpenFile(opts.ReportOptions.Output, os.O_APPEND|os.O_WRONLY, 0644)
+	writer, err := os.OpenFile(opts.ReportOptions.Output, os.O_APPEND|os.O_WRONLY, 0o644)
```

Comment:
what's the difference?

Reply:
No difference, just the golang language server from vscode extension updated it automatically